### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by austinbaccus** [austinbaccus/forza-telemetry#51] [Q&A]
I'm looking for feedback for Forza Telemetry. Ideas, critiques, improvements, and feature requests are all welcome.

**Comment by Bartek-98** [comment]
Great app, exactly what I've looked for!

When it comes to feature requests, maybe a way to change what separator is used in csv? I'm unlucky to live in a country that uses comma as decimal separator and Forza outputs numbers with commas, so the csv is quite hard to use :/

**Reply by austinbaccus** [reply]
A quick fix would be to edit `ForzaCore` > `Program.cs`. Near line 220 is where the `StartNewRecordingSession()` method lives. Inside of that, you could change the delimiter to be `\t` instead of `,` and change the filetype to `.tsv` so that it's tab-separated values instead of comma-separated values.

**Comment by hyacin75** [comment]
Hey - I was going to submit a PR with some changes I thought required to get this to support Xbox, but, it seems to be supporting Xbox just fine out of the box?

I do get an exception when the data starts streaming, but it continues to run and then the app accurately renders the data correlating to what is going on in the game...

I'm guessing the exception is related to my having not set up the loopback stuff, but just sending the data to my system on 6300 from Motorsport (2023) seems to work just fine.

This didn't seem "issue" worthy, and with discussions being so sparse are seemingly not used, I figured throwing it into one that already existed and that you'd get notifications on was a good call.

So the part about "not supported but hoping to implement this soon" in the README is, maybe not right?


Cheers.

**Reply by austinbaccus** [reply]
I honestly haven't played Forza on my Xbox since FM6, so I haven't tested it. 

But if it works, I'm not complaining :D

**Comment by IKeidinI** [comment]
how do i use this on xbox? im so confused since i dont use laptops much
